src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -104,46 +105,41 @@
     <None Remove="keystore\**" />
   </ItemGroup>
   <ItemGroup>
-    <PluginsForBuild Include="$(OutputPath)\Nethermind.Merge.Plugin.dll;
-                      $(OutputPath)\Nethermind.Merge.Plugin.pdb;
-                      $(OutputPath)\Nethermind.Mev.dll;
-                      $(OutputPath)\Nethermind.Mev.pdb;
-                      $(OutputPath)\Nethermind.HealthChecks.dll" />
+    <PluginsForBuild Include="$(OutputPath)\Nethermind.Merge.Plugin.*;
+                              $(OutputPath)\Nethermind.Mev.*;
```

Comment:
Not for this PR, but it will be good to remove plugins (MEV, Merge, HealthChecks) from our Nethermind.Runner.csproj. What do you think?

Reply:
Was thinking about it, but it has 2 issues:
1. Local build, we want to just checkout, build and it should work (plugins)
2. Packaging - we would have to explicitly move it somewhere when we package our release

So while it would be ideal it makes a lot of issues I don't want to dive into now.